test/cypress/integration/widgets/deployments_view_spec.ts

```diff
@@ -265,6 +265,41 @@ describe('Deployments View widget', () => {
             });
         });
 
+        it('should allow to save new filter', () => {
+            const newFilterId = `${filterId}-new`;
+            cy.deleteDeploymentsFilter(newFilterId, { ignoreFailure: true });
+
+            useDeploymentsViewWidget();
+
+            cy.interceptSp('PUT', `/filters/deployments/${newFilterId}`).as('filterCreate');
+
+            cy.contains('button', 'Filter').click();
+            cy.get('.modal').within(() => {
+                cy.contains('.selection', 'Type in values')
```

Comment:
Could we use some more descriptive selector, like:

```suggestion
                cy.contains('.fields', 'Blueprint').contains('.selection', 'Type in values')
```

This will express in the code that we're looking for the Blueprint filter (which could be inferred by the fact that we're putting `blueprintName` there, but I hope the added context does not hurt)